### PR TITLE
Add Leetcode Task Solution

### PR DESCRIPTION
**Pull request by magician1703**

### CODE REVIEW

File: 0013-roman-to-integer/0013-roman-to-integer.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+           result += numbersMap.get(ch) - 2*numbersMap.get(s.charAt(i-1));
+        }
+      
+        else
+            result += numbersMap.get(ch);
+        }
+        
+    return result;
+    }
+}
```
Comment (ArsenEloglian) [inline 0013-roman-to-integer/0013-roman-to-integer.java:27]: please add endline here

**Review by ArsenEloglian — APPROVED** [review]

File: 0013-roman-to-integer/NOTES.md [hunk 2]
```diff
@@ -0,0 +1 @@
+​
```
Comment (ArsenEloglian) [inline 0013-roman-to-integer/NOTES.md:1]: please delete this empty file

**Review by ArsenEloglian — CHANGES_REQUESTED** [review]

**Comment by ArsenEloglian** [comment]
Please create a new folder with your Name and Surname and move your changes there.

**Review by ArsenEloglian — APPROVED** [review]

**Comment by ArsenEloglian** [comment]
@magician1703 
Please merge this PR. (please use squash and merge option)

**Review by WashingtonD — APPROVED** [review]

**Comment by ArsenEloglian** [comment]
@magician1703 
what is the status of this PR?

**Pull request closed without merging**